mkdocs.yml

```diff
@@ -19,10 +19,14 @@ plugins:
 
 # Extensions
 markdown_extensions:
+  - admonition
```

Comment:
Don't include the `admonition` here.

Reply:
Missed to commit `mkdocs.yml` somehow. Removed the offending line.